src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
```

Comment:
better use some class, attached to `body` 😉

Reply:
I can assighn class for the body? :)